test/java/seedu/addressbook/commands/AddCommandTest.java

```diff
@@ -28,7 +28,7 @@
     private static final List<ReadOnlyPerson> EMPTY_PERSON_LIST = Collections.emptyList();
     private static final Set<String> EMPTY_STRING_SET = Collections.emptySet();
 
-    @Test
+    @org.junit.jupiter.api.Test
     public void addCommand_invalidName_throwsException() {
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.